tests/manager/openAccount/managerCanChooseCurrencies.spec.js

```diff
@@ -12,4 +13,15 @@ Test:
 6. Select currency Rupee
 7. Assert the drop-dwon has value Rupee
 */
-});
\ No newline at end of file
+
+const openAccountPage = new OpenAccountPage(page);
+
+await openAccountPage.open();
+await openAccountPage.waitForOpened(); 
+await openAccountPage.selectCurrency('Dollar');  
+await openAccountPage.assertCurrencyDropdownContainsValue('Dollar'); 
+await openAccountPage.selectCurrency('Pound');  
+await openAccountPage.assertCurrencyDropdownContainsValue('Pound'); 
+await openAccountPage.selectCurrency('Rupee');
+await openAccountPage.assertCurrencyDropdownContainsValue('Rupee');
```

Comment:
The method `assertCurrencyDropdownContainsValue` might not correctly verify the selected option in a dropdown. Ensure that the method implementation correctly checks the selected value, possibly by evaluating the dropdown's value attribute directly.